src/main/java/nl/novi/behuiswerkles11/models/Television.java

```diff
@@ -0,0 +1,208 @@
+package nl.novi.behuiswerkles11.models;
+
+import jakarta.persistence.*;
+
+import java.util.List;
+
+@Entity
+@Table(name = "televisions")
+public class Television {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String brand;
+    private String name;
+    private Double price;
+    private Double availableSize;
+    private Double refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private Boolean smartTv;
+    private Boolean wifi;
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    private Boolean ambiLight;
+    private Integer originalStock;
+    private Integer sold;
+    @OneToOne
+    @JoinColumn(name = "remoteControllerId")
+    private RemoteController remoteController;
+    @OneToMany(mappedBy = "television")
```

Comment:
Waarom niet alle relaties in de Television tabel opslaan? Als je kijkt naar het klassendiagram, was dat ook de bedoeling https://github.com/hogeschoolnovi/backend-technisch-ontwerp-klassendiagram-uitwerkingen van de one-to-many relatie